app/representers/notification_preferences_representer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class NotificationPreferencesRepresenter < ThreeScale::Representer
+  include ThreeScale::JSONRepresenter
+
+  property :preferences, as: :notification_preferences
+
+  # NotificationPreferences.available_notifications.each do |notification|
```

Comment:
With this code, the response will look like:

```
{
    "application_created": true,
    "account_created": false,
    "limit_alert_reached_provider": false,
    ...
    "daily_report": false
}
```

I think it's more similar to the request, where we'd just set form-data `application_created=true&account_created=true` etc.

In fact, I think the output is the same as if we were returning `current_user.notification_preferences.preferences` in the controller. But I think going through a representer is more standard and flexible.